sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/QueryPlan.scala

```diff
@@ -442,4 +442,14 @@ object QueryPlan {
         }
     }.canonicalized.asInstanceOf[T]
   }
+
+  /** Normalize and reorder the expressions in the given sequence. */
```

Comment:
Got it. Could you update the above comment, since this is a little bit confusing.